### PR TITLE
feat: Create webhook proxy NGINX Helm chart

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Test Plan
I have tested it on dev cluster with port-forward:
```sh
export POD_NAME=$(kubectl get pods --namespace webhook-proxy -l "app.kubernetes.io/name=webhook-proxy,app.kubernetes.io/instance=webhook-proxy" -o jsonpath="{.items[0].metadata.name}")
export CONTAINER_PORT=$(kubectl get pod --namespace webhook-proxy $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
echo "Visit http://127.0.0.1:8080 to use your application"
kubectl --namespace webhook-proxy port-forward $POD_NAME 8080:$CONTAINER_PORT
```

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.

### CODE REVIEW

**Review by michaeljguarino — COMMENTED** [review]
could we add a small README here just explaining the most basic change needed to deploy against your console, it'd basically be to add:

```yaml
configMap:
  host: https://{your-console-url}
```

**Comment by maciaszczykm** [comment]
@michaeljguarino Would that be enough?

File: charts/webhook-proxy/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+To deploy webhook-proxy against your console edit `values.yaml` to ensure that `configMap.host` points to your console instance:
+
+```yaml
+# Config map with NGINX configuration.
+configMap:
+  # The name of the config map to use.
+  # If not set, a name is generated using the fullname template
+  name: ""
```
Comment (michaeljguarino) [inline charts/webhook-proxy/README.md:8]: i'd drop the name/annotations part here, just set the host

**Review by michaeljguarino — COMMENTED** [review]
lgtm for now i think

File: charts/webhook-proxy/README.md (hunk 1 above)
Comment (maciaszczykm) [reply charts/webhook-proxy/README.md:8]: @michaeljguarino done. If there is anything else you would like to see there let me know. At the moment it's basically standard Helm deployment with config map on top of it.

File: charts/webhook-proxy/templates/configmap.yaml [hunk 2]
```diff
@@ -3,10 +3,6 @@ kind: ConfigMap
 metadata:
   name: {{ include "webhook-proxy.configMapName" . }}
   labels: {{- include "webhook-proxy.labels" . | nindent 4 }}
-  {{- with .Values.configMap.annotations }}
```
[the remark this answers is not in the document]
Comment (maciaszczykm) [reply charts/webhook-proxy/templates/configmap.yaml]: Ok, I misunderstood.

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**